string name
Vector2f p1
Vector2f p2
float32 thickness
